marketdata/rest.go

```diff
@@ -158,22 +163,33 @@ func defaultDo(c *client, req *http.Request) (*http.Response, error) {
 	return resp, nil
 }
 
-func setBaseQuery(q url.Values, start, end time.Time, feed, defaultFeed, asof string) {
-	if !start.IsZero() {
-		q.Set("start", start.Format(time.RFC3339))
+type baseParams struct {
+	Start    time.Time
+	End      time.Time
+	Feed     string
+	AsOf     string
+	Currency string
+}
```

Comment:
Do the members have to be exported when `baseParams` is not?

Reply:
It doesn't really matter.